Apromore-Custom-Plugins/CSVImporter-Portal/src/main/java/org/apromore/plugin/portal/csvimporter/CSVImporterController.java

```diff
@@ -1110,6 +1065,10 @@ private void saveXLog(LogModel logModel, boolean isPublic) {
                 portalContext.refreshContent();
 
             } else {
+
+                if (logModel.getImportLog() == null) {
+                }
+
```

Comment:
Did the body of this condition miss getting filled in?

Reply:
This can be safely removed.